rust/quary-databases/src/databases_postgres.rs

```diff
@@ -864,6 +864,136 @@ models:
         }
     }
 
+    #[tokio::test]
+    async fn postgres_sql_test() {
```

Comment:
nit: test name isn't very descriptive

Reply:
It technically is a "sql test" but you're right it's not obvious right away.